CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ endif()
 
 add_library(${PROJECT_NAME} src/console.cpp)
 target_link_libraries(${PROJECT_NAME} ${Boost_LIBRARIES})
+set_target_properties(${PROJECT_NAME} PROPERTIES SOVERSION 0.3)
```

Comment:
Should we use `${CONSOLE_BRIDGE_MAJOR_VERSION}.${CONSOLE_BRIDGE_MINOR_VERSION}` instead of `0.3`?